### ISSUE TITLE
Bug Report:  Get a book does not contains the book name option and many more bugs...

### ISSUE THREAD

**Issue by Piyusharora2003** [SauravMukherjee44/Aec-Library-Website#1360]
### Reproduction steps

- The helmet of the get book name is Donate books.
-  Get a book does not contains the book name option

![image](https://github.com/SauravMukherjee44/Aec-Library-Website/assets/74443858/e2fec215-97aa-4847-9418-6b463aba4df8)

### Expected behavior

Helmet should be Get a Book

### Actual Behavior

Helmet  is Donate books

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/SauravMukherjee44/Aec-Library-Website/blob/main/CODE_OF_CONDUCT.md)

**Comment by GunjanKMishra** [comment]
Hey, I want to work on this issue.
can you please assign this issue to me!!

**Comment by GunjanKMishra** [comment]
I think this issue is already solved by someone else!!

<a href="https://freeimage.host/i/HPlQnjf"><img src="https://iili.io/HPlQnjf.md.png" alt="HPlQnjf.md.png" border="0"></a>

**Comment by Piyusharora2003** [comment]
> I think this issue is already solved by someone else!!
> 
> [![HPlQnjf.md.png](https://camo.githubusercontent.com/03af56692c8a684292c4cb606ee82ebcf193f4995b744a39902252940ffb7916/68747470733a2f2f69696c692e696f2f48506c516e6a662e6d642e706e67)](https://freeimage.host/i/HPlQnjf)

No thats not the dialogue box for asking for a book , it to contribute moreover, helmet suggest the title at the time, certain jsx is loaded

**Comment by rohanbabbar04** [comment]
Yup good find, just add another option in the navbar as `Get Book`, also add the essential details for getting in the get book form
